verticapy/connection/connect.py

```diff
@@ -66,6 +66,21 @@ def connect(section: str, dsn: Optional[str] = None) -> None:
     dsn: str, optional
         Path to the file containing the credentials.
         If empty, the  Connection File will be used.
+
+    Example
+    -------
+    .. code-block:: python
+
+        from verticapy.connection import *
```

Comment:
please do not use import *, it was old practice and it is now quite dangerous.

Use:
```
from verticapy.connection import available_connections
```
Please also write quick lines.

Ex:
1. Let's look at all the available connections.

2. We can connect using the VerticaDSN connection.

@mail4umar what is the best way to display outputs? Should we use a specific tag: code-block?? Please advice Tyler here.